tests/e2e/src/steps/portal.steps.ts

```diff
@@ -312,6 +319,19 @@ When(
   }
 );
 
+When(
+  "Element {string} should dissapear in {int} seconds",
```

Comment:
updated:
```
  async isElementDissapeared(selecterValue: string, seconds: number) {
    const helper = new Helper(this.world);
    const basePage = new BasePage(this.world);
    const selector = `//*[@${basePage.byTestId}'fee-amount']//*[@alt='${selecterValue}']`;
    const timeout = (seconds + 5) * 1000; // plus extra 5 seconds to avoid false positive timeout issues
    return await helper.checkSelectorHidden(selector, timeout);
  }
```